chat-plugins/datasearch.js

```diff
@@ -675,8 +675,9 @@ function runDexsearch(target, cmd, canAll, message) {
 			}
 
 			if (alts.tiers && Object.keys(alts.tiers).length) {
-				if (alts.tiers[dex[mon].tier]) continue;
-				if (Object.values(alts.tiers).includes(false) && alts.tiers[dex[mon].tier] !== false) continue;
+				let tier = alts.tiers[dex[mon].tier] || alts.tiers[dex[mon].tier].slice(1,11);
```

Comment:
why `.slice(1, 11)`?

Reply:
Also, the second part will never happen, anyway. Or crash. If I'm reading this correctly.

Reply:
That was my typo actually, I guess kris forgot to use the correction

Reply:
It was supposed to be `.slice(1,-1)`

Reply:
yep